app/views/pages/privacy.erb

```diff
@@ -2,51 +2,56 @@
 
 <h1 class="govuk-heading-xl">Privacy notice: how we use your data</h1>
 
-<p>Last updated: 10 April 2024</p>
+<p>Last updated: 22 July 2024</p>
 
 <p>
-  GOV.UK Forms lets UK government teams create online forms to collect
-  information from people and organisations.
+GOV.UK Forms is a platform provided by the Government Digital Service (GDS), which is part of the Cabinet Office. The GOV.UK Forms platform lets UK government teams create online forms to collect information from people and organisations.
```

Comment:
Is it worth taking out the reference to Cabinet Office now rather than later? 😅 

```suggestion
GOV.UK Forms is a platform provided by the Government Digital Service (GDS). The GOV.UK Forms platform lets UK government teams create online forms to collect information from people and organisations.
```

Reply:
I think we need to say who the legal entity is, so suggest we stick with CO for now. It's the sort of thing we'd want to do on a x-GDS basis, anyway.